README.md

```diff
@@ -95,6 +95,18 @@ analysis-runner \
 For more examples (including for running an R script and dataproc), see the
 [examples](examples) directory.
 
+### GitHub Authorisation
```

Comment:
```suggestion
### GitHub Authentication
```

I think so anyway… who can keep the distinction between these two straight… :smile:

Reply:
The various readmes were inconsistent with `Authorisation` vs`Authorization`, so this solves that problem too 🤷